lib/hako/scripts/create_log_group.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'aws-sdk'
+require 'hako'
+require 'hako/script'
+
+module Hako
+  module Scripts
+    class CreateLogGroup < Script
+      # @param [Hash<String, Container>] containers
+      # @return [nil]
+      def deploy_starting(containers)
+        containers.each do |_, container|
+          next unless container.definition.key?('log_configuration')
```

Comment:
Thanks for pointing out.
Fixed it.

https://github.com/eagletmt/hako/pull/22/commits/8dba1c11a1b57d82e4537dc0b400da45a9b7f509